OWNERS_ALIASES

```diff
@@ -7,4 +7,15 @@ aliases:
     - lucab
     - wking
     - vrutkovs
+  app-sre:
```

Comment:
Would be nice if this could be defined in a common place and pulled in from there. [The OWNERS doc](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md) doesn't talk about such a feature though.